src/main/resources/license-data.json

```diff
@@ -118,6 +118,14 @@
       "license": "Apache-2.0",
       "licenseSource": "data"
     },
+    "software.amazon.awssdk.marketplacemetering": {
```

Comment:
Normally we follows order in this file. So I would suggest to move this below where package that starts with "s" from "com."